### ISSUE TITLE
Invalid config files: Template fetching failed: Failed to find imported sample_quickstart_existing_network.yaml 

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-google-gdm-templates-v2#9]
## Describe the bug
 
The "type" on the last line of sample_quickstart_existing_network.yaml is pointing to the new stack python file, quickstart.py. As a result it errors.

https://github.com/F5Networks/f5-google-gdm-templates-v2/blob/main/examples/quickstart/sample_quickstart_existing_network.yaml
 
## Expected behavior
 
Deployment should proceed with an existing stack. YAML config should call the correct python template.

Good logs...
```
gcloud deployment-manager deployments create ${DEPLOYMENT_NAME} --config ${CONFIG_FILE}
The fingerprint of the deployment is b'PWlw8iOMYym1Q8RWv-odaQ=='
Waiting for create [operation-1650999139817-5dd933140d6e2-a3d287fc-8ffa239f]...done.              
Create operation operation-1650999139817-5dd933140d6e2-a3d287fc-8ffa239f completed successfully.
NAME                         TYPE                       STATE      ERRORS  INTENT
giroux321-app-int-fw         compute.v1.firewall        COMPLETED  []
giroux321-app-vip-fw         compute.v1.firewall        COMPLETED  []
giroux321-bigip-ti           compute.v1.targetInstance  COMPLETED  []
giroux321-bigip1             compute.v1.instance        COMPLETED  []
giroux321-fwrule1            compute.v1.forwardingRule  COMPLETED  []
giroux321-http-healthcheck   compute.v1.healthCheck     COMPLETED  []
giroux321-https-healthcheck  compute.v1.healthCheck     COMPLETED  []
giroux321-mgmt-fw            compute.v1.firewall        COMPLETED  []
giroux321-tcp-healthcheck    compute.v1.healthCheck     COMPLETED  []
```
 
## Current behavior
 
The yaml file is pointing to the incorrect file, so an import error occurs.

Currently...
```
    type: quickstart.py
```

Errors otherwise...
```
ERROR: (gcloud.deployment-manager.deployments.create) Error in Operation [operation-1650998946779-5dd9325bf51cc-f826ca86-bac91560]: errors:
- code: INVALID_CONFIG
  message: "Invalid config files: Template fetching failed: Failed to find imported\
    \ template 'quickstart.py' for resource {name=quickstart-existing-network-py,\
    \ properties={bigIpImageName=f5-bigip-16-1-2-1-0-0-10-payg-best-plus-1gbps-211222210245,\
    \ bigIpInstanceType=n1-standard-8, bigIpRuntimeInitConfig=https://raw.githubusercontent.com/F5Networks/f5-google-gdm-templates-v2/v2.0.0.0/examples/quickstart/bigip-configurations/runtime-init-conf-3nic-payg-with-app.yaml,\
    \ networks={externalNetworkName=jgiroux-net-ext, internalNetworkName=jgiroux-net-int,\
    \ mgmtNetworkName=jgiroux-net-mgmt}, numNics=3, owner=giroux, provisionPublicIp=true,\
    \ region=us-west1, restrictedSrcAddressApp=[0.0.0.0/0], restrictedSrcAddressMgmt=[0.0.0.0/0],\
    \ subnets={appSubnetName=jgiroux-subnet-ext, internalSubnetName=jgiroux-subnet-int,\
    \ mgmtSubnetName=jgiroux-subnet-mgmt}, uniqueString=giroux321, zone=us-west1-a},\
    \ type=quickstart.py}.."
```
 
## Possible solution
 
Needs to be...
```
    type: quickstart-existing-network.py
```
 
## Steps to reproduce
 
1. Run template for existing stack quickstart example
 
## Screenshots
 
n/a
 
## Context
 
n/a
 
## Your Environment
 
n/a

**Comment by mikeshimkus** [comment]
Thanks @JeffGiroux. We addressed this in internal issue 2957 and it should be included in the next release.

**Comment by shyawnkarim** [comment]
Closing.

This issue was resolved with [Release 2.6.0.0](https://github.com/F5Networks/f5-google-gdm-templates-v2/releases/tag/v2.6.0.0).
